sources/packages/api/src/database/entities/disbursement-values.model.ts

```diff
@@ -39,12 +39,14 @@ export class DisbursementValue extends RecordDataModel {
   valueCode: string;
   /**
    * Amount of money to be disbursed for one particular loan/grant.
+   * !Decimal values are retrieved by Typeorm as string from Postgres.
```

Comment:
Oh. that is surprising. thanks for adding the comment.

Reply:
Yes, to me too. Anything that the javascript number is not able to support, usually due to the maximum amount (not decimals) is going to be mapped to a string like, "bigint", "numeric", etc.